settings.js

```diff
@@ -100,6 +100,22 @@ var settings = {
 	commandMappings: path.join(__dirname, 'mappings.json')
 };
 
+function envValue(varName, defaultValue) {
+	var value = process.env[varName];
+	return (typeof value === 'undefined') ? defaultValue : value;
+}
+
+function envValueBoolean(varName, defaultValue) {
+	var value = envValue(varName);
```

Comment:
No, this is on purpose to get the raw value. The default handing is in `envValueBoolean`